layouts/soarian-financials_stu3_sidebar.html

```diff
@@ -0,0 +1,59 @@
+<div id="js-sidebar" class="sidebar-shell">
+  <div class="js-toggle-list sidebar-module expandable">
+    <ul>
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/soarian-financials/stu3/">Overview</a>
+        </h3>
+        <ul class="js-guides">
+          <li><a href="/authorization/">Authorization</a></li>
+<!--          <li><a href="/soarian-financials/stu3/conformance/">Conformance Metadata</a></li>
+          <li><a href="/soarian-financials/stu3/operation-definition/">OperationDefinition</a></li>
+
+          <li><a href="/soarian-financials/stu3/structure-definition/">StructureDefinition</a></li>
+-->
+        </ul>
+      </li>
+<!--
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/soarian-financials/stu3/encounters/">Encounters</a>
+        </h3>
+        <ul class="js-guides">
+          <li><a href="/soarian-financials/stu3/encounters/encounter/">Encounter</a></li>
+        </ul>
+      </li>
+
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/soarian-financials/stu3/financials/">Financials</a>
+        </h3>
+        <ul class="js-guides">
+          <li><a href="/soarian-financials/stu3/financials/account/">Account</a></li>
+          <li><a href="/soarian-financials/stu3/financials/coverage/">Coverage</a></li>
+        </ul>
+      </li>
+-->
+      <li class="js-topic">
+        <h3><a href="#" class="js-expand-btn collapsed arrow-btn" data-proofer-ignore></a><a href="/soarian-financials/stu3/individuals/">Individuals</a>
+        </h3>
+        <ul class="js-guides">
+          <li><a href="/soarian-financials/stu3/individuals/patient/">Patient</a></li>
+          <!--<li><a href="/soarian-financials/stu3/individuals/related-person/">RelatedPerson</a></li>-->
```

Comment:
related-person.md exists so this should be uncommented

Reply:
We are not ready to expose this resource yet